DP-16314 [a11y] Campaign landing page feature (#816)
massgov/mayflower: changelogs/DP-16314.md

Minor
Changed

> - (Patternlab) [Features] DP-16314: Correct closing heading level for feature card. #816

Reviewer: ```suggestion
- (Patternlab) [Card] DP-16314: Correct closing heading level for feature card. #816
```

> - (Patternlab) [Features] DP-16314: Remove id from secondary cards. #816

Reviewer: ```suggestion
- (Patternlab) [Card] DP-16314: Remove id from secondary cards. #816
```

> - (Patternlab) [Features] DP-16314: Update landing page jsons to match the change for ID. #816

Reviewer: ```suggestion
- (Patternlab) [Pages Marketing Campaign] DP-16314: Update landing page jsons to match the change for ID. #816
```

Added

> - (Patternlab) [Features] DP-16314: Add focusable='false' to svg icons. #816

Reviewer: ```suggestion
- (Patternlab) [SVG Icons Arrow] DP-16314: Add focusable='false' to svg icons. #816
```

Fixed

> - (Patternlab) [Features] DP-16314: Correct the wrong class name for card and the condition to add the class to the contaienr. #816

Reviewer: ```suggestion
- (Patternlab) [Card] DP-16314: Correct the wrong class name for the card and the condition to add the class to the container. #816
```
